srcpkgs/clthreads/template

```diff
@@ -1,23 +1,19 @@
 # Template file for 'clthreads'
 pkgname=clthreads
 version=2.4.2
-revision=1
+revision=2
 build_wrksrc=source
 build_style=gnu-makefile
 make_install_args="LIBDIR=/usr/lib"
 short_desc="C++ wrapper library around the POSIX threads API"
-maintainer="bluntphenomena <gregwyd@gmail.com>"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
 license="LGPL-2.1-or-later"
 homepage="https://kokkinizita.linuxaudio.org/linuxaudio/"
 distfiles="https://kokkinizita.linuxaudio.org/linuxaudio/downloads/${pkgname}-${version}.tar.bz2"
 checksum=c659b14c0c4055c44432cb83060b95d30ae0c1ecc6f50d73968e239c100f7a31
 
 CXXFLAGS="-fPIC"
 
-pre_build() {
-	vsed -i Makefile -e '/march=native/d'
-}
```

Comment:
I assume they removed `-march=native` from their own makefiles?

Reply:
no but i find it misleading to keep these patches since it doesnt affect the build process (look my initial comment on this PR)

Reply:
same as https://github.com/void-linux/void-packages/commit/c9f3bfc141fe09072d0339417f808faacd3a70f6

Reply:
Sure, I was missing that piece of context, since the vsed doesn't make it clear it was dealing with those flags :)

Reply:
So, the change won't do anything? I prefer to left it there.

Reply:
It avoids carrying changes which are no longer necessary...

Reply:
I agree @ericonr. I think part of the purpose of https://github.com/void-linux/void-packages/commit/2325ae153bd6011682782a6712f87338a9747ac7 was to eliminate patches like these?